tests/integration/api_org_test.go

```diff
@@ -144,9 +144,7 @@ func TestAPIOrgDeny(t *testing.T) {
 func TestAPIGetAll(t *testing.T) {
 	defer tests.PrepareTestEnv(t)()
 
-	token := getUserToken(t, "user1", auth_model.AccessTokenScopeReadOrg)
-
-	req := NewRequestf(t, "GET", "/api/v1/orgs?token=%s", token)
+	req := NewRequestf(t, "GET", "/api/v1/orgs") // token not required
```

Comment:
It is required for limited and private repos.
Perhaps we should add a test case for that.

Reply:
Do you mean it should return public repo when accessing without a token, and return all repos when using a token with correct scope?
That will complicate things a lot, and many fixes are needed.